Source/KerbalCombatSystems/EscapePod.cs

```diff
@@ -44,16 +49,42 @@ public void ManualEscape(KSPActionParam param)
             BeginEscape();
         }
 
+        public override void OnStart(StartState state)
+        {
+            if (!HighLogic.LoadedSceneIsFlight) return;
+
+            //create the appropriate lists
+            shipControllerList = new List<Part>();
+            List<ModuleShipController> AIModulesList;
+
+            //designate ship that's being escaped from
+            parent = vessel;
+
+            //find ai parts and add to list
+            AIModulesList = vessel.FindPartModulesImplementing<ModuleShipController>();
+            foreach (var Controller in AIModulesList)
+            {
+                shipControllerList.Add(Controller.part);
+            }
+
+            StartCoroutine(StatusRoutine());
+        }
+
         private IEnumerator RunEscapeSequence()
         {
             //stop the status checker temporarily
             StopCoroutine(StatusRoutine());
 
             // try to pop decoupler
-            if (seperator != null)
+            try
```

Comment:
A failed try-catch here creates an error in the log. If the expectation is that it is acceptable for seperator to be null then I think it should be an if-then so we don't create an error. Pretty sure the code path would be the same but without the error.

Reply:
Concern is if the decoupler exists but already fired will it throw up an error with the current decouple subfunction